src/components/externalLinks/ExternalLinksCRUD/IdentifierSelector.tsx

```diff
@@ -30,12 +31,12 @@ export default function IdentifierSelector({
                             <div className="flex left pl-8 pt-6 pb-6" onClick={markOptionAsExternalApp}>
                                 <AddIcon className="icon-dim-16 fcb-5 mr-8" />
                                 <span className="fs-13 fw-4 lh-20 cb-5">
-                                    External helm app ‘{identifierSearchInput}’
+                                    {`${isFullMode ? 'External helm' : 'Helm'} app ‘${identifierSearchInput}’`}
                                 </span>
                             </div>
                         ) : (
                             <div className="cn-5 pl-8 pt-6 pb-6 dc__italic-font-style">
-                                Enter app name for externally deployed helm apps
+                                {`Enter app name for${isFullMode ? ' externally ' : ' '}deployed helm apps`}
```

Comment:
Please Replace use of ternary with &&

Reply:
Can't use it as it'll append false to the string when the isFullMode is false.